renderdoc/driver/vulkan/wrappers/vk_get_funcs.cpp

```diff
@@ -893,21 +839,7 @@ void WrappedVulkan::vkGetPhysicalDeviceProperties2(VkPhysicalDevice physicalDevi
 {
   ObjDisp(physicalDevice)->GetPhysicalDeviceProperties2(Unwrap(physicalDevice), pProperties);
 
-  MakeFakeUUID();
-
-  memcpy(pProperties->properties.pipelineCacheUUID, fakeRenderDocUUID, VK_UUID_SIZE);
-
   ClampPhysDevAPIVersion(&pProperties->properties, physicalDevice);
-
-  // internal RenderDoc UUID for shader object binary
-  VkPhysicalDeviceShaderObjectPropertiesEXT *shadObj =
-      (VkPhysicalDeviceShaderObjectPropertiesEXT *)FindNextStruct(
-          pProperties, VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_OBJECT_PROPERTIES_EXT);
-
-  if(shadObj)
-  {
-    memcpy(shadObj->shaderBinaryUUID, fakeRenderDocUUID, VK_UUID_SIZE);
```

Comment:
We need to keep the fake UUID for the shader EXT because unlike pipeline caches it _does_ allow omitting of SPIR-V and passing pure binaries. We block this later at creation time but it would be invalid to refuse to load shaders that otherwise match the UUID.

Reply:
good point I was unaware of that, thanks.